roles/neutron-common/templates/etc/neutron/rootwrap.conf

```diff
@@ -12,9 +12,9 @@ filters_path=/etc/neutron/rootwrap.d,/usr/share/neutron/rootwrap
 # If not specified, defaults to system PATH environment variable.
 # These directories MUST all be only writeable by root !
 {% if openstack_install_method == "package" -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin,{{ 'neutron'|ursula_package_path(openstack_package_version) }}/bin
 {% else -%}
-exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin
+exec_dirs=/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin
```

Comment:
Ok... I have already tested l3ha with /usr/local/bin in exec_dirs for both source and package based installation. The neutron-keepalived-state-change process is started fine when router is created.